osu.Game/Beatmaps/BeatmapStore.cs

```diff
@@ -89,15 +89,13 @@ protected override void Purge(List<BeatmapSetInfo> items, OsuDbContext context)
 
         public IQueryable<BeatmapSetInfo> BeatmapSetsOverview => ContextFactory.Get().BeatmapSetInfo
                                                                                .Include(s => s.Metadata)
-                                                                               .Include(s => s.Beatmaps)
-                                                                               .AsNoTracking();
```

Comment:
why re-enable tracking? this will have negative performance implications. we don't really want entity tracking in most cases, especially not this one.

Reply:
This can be added back.